gateway/config/config.go

```diff
@@ -69,6 +69,7 @@ type Crud map[string]*CrudStub // The key here is the alias for database type
 // CrudStub holds the config at the database level
 type CrudStub struct {
 	Type            string                    `json:"type,omitempty" yaml:"type"` // database type
+	Name            string                    `json:"name,omitempty" yaml:"name"` // name of the logical database or schema name according to the database type
```

Comment:
We should call this field `database` or `dbName`

Reply:
i 'll rename it to dbName,
i had given a generic name as it can be db name or schema name.